docs/logging.asciidoc

```diff
@@ -1,34 +1,36 @@
-[[log-correlation]]
-== Log correlation
+[[logs]]
+== Logs
 
-Log correlation allows you to navigate to all logs belonging to a particular trace, and vice-versa -- for a specific log, see in which context it has been logged, and which parameters the user provided.
+Elastic Python APM Agent provides the following log features:
 
-The Agent provides integrations with both the default Python logging library,
-as well as http://www.structlog.org/en/stable/[`structlog`].
+- <<log-correlation-ids>> : Automatically inject correlation IDs that allow navigation between logs, traces and services.
+- <<log-reformatting>> : Automatically reformat plaintext logs in {ecs-logging-ref}/intro.html[ECS logging] format.
 
-* <<logging-integrations>>
-* <<log-correlation-in-es>>
+Those features are part of {observability-guide}/application-logs.html[Application log ingestion strategies].
+
+The {ecs-logging-python-ref}/intro.html[`ecs-logging-python`] library can also be used to use the {ecs-logging-ref}/intro.html[ECS logging] format without an APM agent.
+When deployed with the Python APM agent, the agent will provide <<log-correlation-ids,log correlation>> IDs.
 
 [float]
-[[ecs-logging]]
```

Comment:
This is a breaking change, but I suppose it's probably worth standardizing the links between agents.